media/css/mozorg/home/home-2018.scss

```diff
@@ -79,136 +77,83 @@ h4 {
-        padding-bottom: 40px;
+        padding-bottom: $layout-sm;
     }
 
     .c-primary-cta-logo {
-        background-position: top left;
-        margin-bottom: $spacing-2xl;
-        padding-left: $spacing-2xl;
-        padding-top: 0;
-        text-align: left;
-        line-height: 40px;
-    }
-
-    .download-firefox-primary-cta .mzp-l-content {
-        @include at2x('/media/img/home/2018/banner-fffy.png', 720px, 350px);
-        background-position: center bottom 30px;
-        padding-bottom: 350px;
-    }
-
-    .fxaccount-primary-cta .mzp-l-content {
-        @include at2x('/media/img/firefox/accounts/fxa-hero-img.png', 584px, 389px);
-        padding-bottom: 390px;
-    }
-}
-
-@media #{$mq-sm} {
-    .download-firefox-primary-cta {
-        background: #bddefd;
-
-        .mzp-l-content {
-            background: url('/media/img/home/2018/banner-fffy.png') center bottom 10px no-repeat;
-            @include background-size(360px 175px);
-            padding-bottom: 180px;
-
-            .c-primary-cta-title-sub {
-                text-align: left;
-            }
-        }
-    }
-
-    .fxaccount-primary-cta {
-        background: $color-white;
-
-        .mzp-l-content {
-            background: url('/media/img/firefox/accounts/fxa-hero-img.png') center bottom no-repeat;
-            @include background-size(360px 240px);
-            padding-bottom: 250px;
-        }
+        margin-bottom: $spacing-xl;
     }
 }
 
 @media #{$mq-lg} {
     .c-primary-cta {
         @include bidi(((text-align, left, right),));
-        border-top: 100px solid #fff;
+        border-top: 100px solid $color-white; // to hide sticky CTA
         margin-bottom: -100px;
         position: relative;
         top: -100px;
 
         .c-primary-cta-wrapper {
             @include border-box;
-            max-width: 520px;
+            max-width: 50%;
         }
 
         .c-primary-cta-title-sub {
             margin: 0;
         }
     }
 
-    .download-firefox-primary-cta .mzp-l-content {
-        @include at2x('/media/img/home/2018/banner-fffy.png', 720px, 350px);
-        background-position: 580px center;
-        padding-bottom: $spacing-lg;
+    .download-firefox-primary-cta {
+        background-image: url('/media/img/home/2018/browser.png'), url('/media/img/home/2018/bg-primary.svg');
+        background-size: 400px 278px, auto;
+        background-position: calc(50vw + 34px) 60px, calc(50vw - 41px) -78px;
+        background-repeat: no-repeat;
+
+        @media #{$mq-high-res} {
+            background-image: url('/media/img/home/2018/browser-high-res.png'), url('/media/img/home/2018/bg-primary.svg');
+        }
     }
 
-    .fxaccount-primary-cta .mzp-l-content {
-        @include at2x('/media/img/firefox/accounts/fxa-hero-img.png', 584px, 389px);
-        background-position: 560px bottom;
-        padding-bottom: $spacing-lg;
+    .fxaccount-primary-cta {
+        @include at2x('/media/img/home/2018/home.jpg', 815px, 362px);
```

Comment:
This image is 404, the file type should be .png (though might jpg be a smaller file? Does it need transparency?)

Reply:
I converted these to .jpg like I intended to and cropped out some white space so it appears a bit bigger now.